### ISSUE TITLE
Old branches to be deleted or PR opened and PR's to be refreshed or deleted

### ISSUE THREAD

**Issue by valeriupredoi** [ESMValGroup/ESMValCore#459]
Assigning people to have a look and delete all the branches that don't have an active PR or do but the PR is ancient; please mark KEEP if you choose not to delete, tick the box after you deleted otherwise; I am also listing the ones that need serious refreshing :beer:

- [ ] https://github.com/ESMValGroup/ESMValCore/tree/activity_alias @jvegasbsc DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/add_clip @bascrezee DELETE OLD
- https://github.com/ESMValGroup/ESMValCore/pull/247 @jvegasbsc REFRESH(conflicts)
- https://github.com/ESMValGroup/ESMValCore/pull/300 @jvegasbsc REFRESH(conflicts)
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/check_aux_coords @jvegasbsc DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/dev_add_activity_badc @jvegasbsc DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/dev_append @bjoernbroetz DELETE NoPR
- https://github.com/ESMValGroup/ESMValCore/pull/166 @jvegasbsc REFRESH
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/dev_default_preproc @jvegasbsc DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/dev_emac_import @schlunma DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/dev_sea_ice_speed @jvegasbsc DELETE NoPR
- https://github.com/ESMValGroup/ESMValCore/pull/274 @bouweandela REFRESH(conflicts)
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/development_Zonal_Meridional_statistics_24 @ledm DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/development_amoc_cmip6 @ledm DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/development_cmip6_Ofx_Omon_fix @ledm  DELETE NoPR fixed by #439 ?
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/development_fix_CMIP6_CV @ledm DELETE NoPR
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/development_fx_RESTRUCTURED_Zonal_Meridional_statistics @ledm DELETE NoPR
- https://github.com/ESMValGroup/ESMValCore/pull/214 @ledm REFRESH(conflicts) superseded by #439 
- https://github.com/ESMValGroup/ESMValCore/pull/270 @ledm REFRESH(conflicts)
- [ ] https://github.com/ESMValGroup/ESMValCore/tree/development_ocean_development_for_v2_paper_1166 @ledm DELTE NoPR

(to be continued tomorrow, am tired now)

**Comment by mattiarighi** [comment]
Spring cleaning!

**Comment by valeriupredoi** [comment]
Indeed, found something useful to do instead of starting to write a gazillion slides for ARCDAP-3 :grin:

**Comment by valeriupredoi** [comment]
OK completed the list - the winner is hands down @jvegasbsc - we can call Javi Ernest Hemingway from now on given that the guy had more notes to fit in a hundred unpublished novels :grin:

**Comment by tomaslovato** [comment]
@valeriupredoi is this 2020 cleaning activity still a valid issue?

**Comment by valeriupredoi** [comment]
we now have the fancy stale Issue/PR policeman but that doesn't take care of stale historical branches! I am going to delete those stale branches myself - what do @ESMValGroup/technical-lead-development-team @ESMValGroup/userengagementteam think of that? 🍻
